### ISSUE TITLE
Partner Agency Orders

### ISSUE THREAD

**Issue by katiesteiner** [rubyforgood/diaperpartner#2]
As a partner agency user I'd like to be able to submit an order to the diaperbase for the amount of diapers I need for the following month by size.

Diaper sizes should follow the inventory list from Diaperbase and have the option to list sizes not available currently in the inventory list.

**Megan/Sean: Would that realistically work or would you prefer the agency only be able to request items that are currently available in the Diaperbase inventory list?
**Sean: Is there a way to dynamically connect Diaperbase and the Partner Agency Portal? If not, we may need a master list of size options to include on this order form.

**Comment by seanmarcia** [comment]
I think they should be able to request regardless if they have the inventory. That way if they acquire the diapers before the distribution date they can still choose to fill it.

**Comment by SCDB** [comment]
I agree with Sean--I would want my agencies to still be able to order diapers even if we didn't currently have the size in stock. We are often getting deliveries and donations up till the day before an agency is coming to pick up their order.
